src/reducers/books.js

```diff
@@ -0,0 +1,23 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = {
+  books: [
+    { bookId: 1, title: 'Understanding in C', category: 'Programming' },
```

Comment:
This implementation has moved from bad to worst in my opinion 😌. You don't want to make users enter manually the id of the book they are entering. This would most like lead to a duplicate id being created since the user may not know the last id of the book in the list😌. I gave suggestions in the earlier review as to how you could handle this, kindly check it out and apply it else if you end up having a duplicate key the virtual dom will crush we have conflicting key attributes. packages like [`short-id`](https://www.npmjs.com/package/shortid) or [`UUID`](https://www.npmjs.com/package/uuid) have a way of making sure your ids are unique.
`{ bookId: shortid.generate(), title: 'Understanding in C', category: 'Programming' },`

Reply:
It was not necessary to do that in this milestone, that's why I have included it manually. But looks like it's mandatory according to you, I have included the UUID for the random number generator.

Thank you for the suggestion and guidance in this code review. I've got to learn many things with this and previous code review.